pgjdbc/src/main/java/org/postgresql/jdbc/ResourceLock.java

```diff
@@ -0,0 +1,34 @@
+package org.postgresql.jdbc;
```

Comment:
Needs license

Reply:
License added + code adjusted to use condition await() and signalAll() - ready to re-run the test workflows